components/form/UnitInput.vue

```diff
@@ -188,6 +188,7 @@ export default {
     :tooltip-key="tooltipKey"
     :required="required"
     :placeholder="placeholder"
+    :hide-arrows="true"
```

Comment:
I intended to only hide it for the auto-replace option because the unit overlaps with the arrows in this case, but not in other cases where the unit input is used.

Reply:
In that case, we might need to introduce a prop that we can pass to `UnitInput` in order to hide the stepper. The current implementation with `:hide-arrows="true"` changes other forms across the dashboard, for example:

1. Open the main menu and select a cluster
2. Go to Cluster => Projects/Namespace => Create Project
3. Click on Resource Quotas => Add Resource 
4. Select CPU Reservation from the dropdown

### Expected
We would expect to have the stepper for the limits

![image](https://user-images.githubusercontent.com/835961/153237778-613fc152-977f-4ce2-9805-c79cfa2f16a6.png)

### Actual
This PR removes the stepper for the limits

![image](https://user-images.githubusercontent.com/835961/153237881-76276760-1738-4ccb-92a4-a07764b8c9e4.png)

Reply:
Good catch. I have updated the UnitInput so that it takes a prop for hideArrows, so that that is controlled from the form. Note: In Chrome, the arrows are hidden until you mouse over the inputs, whereas in Firefox, you see the arrows as soon as the page loads.